model-optimizer/extensions/front/HSigmoid_fusion.py

```diff
@@ -0,0 +1,111 @@
+
+    rename_nodes([(mul_2, mul_2_name + '/TBR'), (hsigmoid, mul_2_name)])
+
+
+class HSigmoidWithClamp(FrontReplacementSubgraph):
+    """
+    The transformation looks for the pattern with ReLU6 (Clamp) defining the HSigmoid function:
+    HSigmoid(x) = Relu6(x + 3) / 6.0.
+    """
+    enabled = True
+
+    def run_after(self):
+        return [AttributedClampNormalizer]
+
+    def pattern(self):
+        return dict(
+            nodes=[
+                ('input', dict()),
+                ('add', dict(op='Add')),
+                ('const_0', dict(op='Const', value=lambda v: v is not None and np.allclose(v, 0.0, atol=1e-6))),
+                ('const_3', dict(op='Const', value=lambda v: v is not None and np.allclose(v, 3.0, atol=1e-6))),
+                ('const_6', dict(op='Const', value=lambda v: v is not None and np.allclose(v, 6.0, atol=1e-6))),
+                ('const_1_6', dict(op='Const', value=lambda v: v is not None and np.allclose(v, 1 / 6.0, atol=1e-6))),
+                ('clamp', dict(op='Clamp')),
+                ('mul_2', dict(op='Mul')),
+            ],
+            edges=[
+                ('input', 'add', {}),
+                ('const_3', 'add', {}),
+                ('add', 'clamp', {'in': 0}),
+                ('const_0', 'clamp', {'in': 1}),
+                ('const_6', 'clamp', {'in': 2}),
+                ('clamp', 'mul_2', {}),
+                ('const_1_6', 'mul_2', {}),
+            ])
+
+    def replace_sub_graph(self, graph: Graph, match: [dict, SubgraphMatch]):
+        replace_with_hsigmoid(graph, match)
+
+
+class HSigmoidWithMinMax(FrontReplacementSubgraph):
+    """
+    The transformation looks for the pattern with Min/Max defining the HSigmoid function:
+    HSigmoid(x) = Min(Max(x + 3, 0), 6) / 6.0.
+    """
+    enabled = True
+
+    def run_after(self):
+        return [AttributedClampNormalizer]
+
+    def pattern(self):
+        return dict(
+            nodes=[
+                ('input', dict()),
+                ('add', dict(op='Add')),
+                ('const_0', dict(op='Const', value=lambda v: v is not None and np.allclose(v, 0.0, atol=1e-6))),
+                ('const_3', dict(op='Const', value=lambda v: v is not None and np.allclose(v, 3.0, atol=1e-6))),
+                ('const_6', dict(op='Const', value=lambda v: v is not None and np.allclose(v, 6.0, atol=1e-6))),
+                ('const_1_6', dict(op='Const', value=lambda v: v is not None and np.allclose(v, 1 / 6.0, atol=1e-6))),
+                ('max', dict(op='Maximum')),
+                ('min', dict(op='Minimum')),
+                ('mul_2', dict(op='Mul')),
+            ],
+            edges=[
+                ('input', 'add', {'out': 0}),
+                ('const_3', 'add', {}),
+                ('add', 'max', {}),
+                ('const_0', 'max', {}),
+                ('max', 'min', {}),
+                ('const_6', 'min', {}),
+                ('min', 'mul_2', {}),
+                ('const_1_6', 'mul_2', {}),
+            ])
+
```

Comment:
Why don't you add other fusion transformation where `Divide` is used instead of `Multiply` in case division by 6.0